CDCgov/prime-reportstream
# Add availability testing for the SFTP container

This PR adds availability testing for the SFTP container. A new function in the admin app does the checking, which then feeds back into Application Insights as an additional availability check. 

Test Steps:
1. Do `terraform apply`. Ensure Application Insights display status for the SFTP container. 

## Changes
- A new admin app function: `confirm_sftp_access.py`.
- A new availability test in App Insights that uses this function to poll SFTP.

## Checklist

### Testing
## Linked Issues
- Fixes #10368
